test/lib/Elastica/Test/ResponseTest.php

```diff
@@ -201,6 +201,23 @@ public function testIsNotOkBulkItemsWithStatusField()
     }
 
     /**
+     * @group unit
+     */
+    public function testDecodeResponseWithBigIntSetToTrue()
+    {
+        $response = new Response(json_encode(array(
+            'took' => 213,
+            'items' => array(
+                array('index' => array('_index' => 'rohlik', '_type' => 'grocery', '_id' => '707891', '_version' => 4, 'status' => 200)),
+                array('index' => array('_index' => 'rohlik', '_type' => 'grocery', '_id' => '707893', '_version' => 4, 'status' => 200)),
+            ),
+        )));
+        $response->setJsonBigintConversion(true);
+
+        $this->assertTrue(is_array($response->getData()));
```

Comment:
I'm fine with the test. But I assume this test would also have passed without the code changes you made? It is more for me to understand if we test if the bug was actually fixed or if we test that we can use the function without any side affects?